specs/form-associated-custom-element.md

```diff
@@ -0,0 +1,115 @@
+# Form Associated Custom Elements
+
+## Overview
+This spec regards a general implementation to facilitate form communication and connection for components that should be form-associated, including checkboxes,
+radios, text inputs, and other components which store user input and should be captured during form submission.
```

Comment:
Looks like there is a return here between checkboxes and radios.

Reply:
fixed.